toolkit/templates/documents.html

```diff
@@ -7,6 +7,7 @@
         {% if item.analytics_category %}data-analytics-category="{{ item.analytics_category }}" {% endif %}
         {% if item.analytics_action %}data-analytics-action="{{ item.analytics_action }}" {% endif %}
         {% if item.analytics_label %}data-analytics-label="{{ item.analytics_label }}" {% endif %}
+        {% if item.analytics_target %} data-analytics-target-selector="{{ item.analytics_target }}" {% endif %}
```

Comment:
You have acquired an extra space that is going to annoy those of us with OCD tendencies :)